src/screenbot.js

```diff
@@ -38,6 +38,21 @@ var Screenbot = (function Screenbot() {
       xhttp.send();
     };
 
+    var _handleError = function(args, cb){
+      var err = new ScreenbotError(args);
+      return cb(err);
+    };
+
+    var ScreenbotError = function(args){
+      this.name = 'Screenbot Error';
+      this.message = args.message || 'An error has occured';
+      this.code = args.code;
+      this.stack = (new Error()).stack;
```

Comment:
Should be able to remove this line and as the first line in the function use `Error.call(this)` to call the superclass constructor.